code-of-conduct-languages/ma.md

```diff
@@ -0,0 +1,52 @@
+## CNCF सामुदायिक आचार संहिता v1.3
+
+अन्य भाषा उपलब्ध:
+- [अरबी/العربية](code-of-conduct-languages/ar.md)
+- [बल्गेरियाई/Български](code-of-conduct-languages/bg.md)
+- [चीनी/中文](code-of-conduct-languages/zh.md)
+- [चेक/Česky](code-of-conduct-languages/cs.md)
+- [फ़ारसी/فارسی](code-of-conduct-languages/fa.md)
+- [फ़्रेंच/Français](code-of-conduct-languages/fr.md)
+- [जर्मन/Deutsch](code-of-conduct-languages/de.md)
+- [इंडोनेशियाई/Bahasa Indonesia](code-of-conduct-languages/id.md)
+- [इतालवी/Italiano](code-of-conduct-languages/it.md)
+- [जापानी/日本語](code-of-conduct-languages/jp.md)
+- [कोरियाई/한국어](code-of-conduct-languages/ko.md)
+- [पोलिश/Polski](code-of-conduct-languages/pl.md)
+- [पुर्तगाली/Português](code-of-conduct-languages/pt.md)
+- [रूसी/Русский](code-of-conduct-languages/ru.md)
+- [स्पेनिश/Español](code-of-conduct-languages/es.md)
+- [तुर्की/Türkçe](code-of-conduct-languages/tr.md)
+- [यूक्रेनी/Українська](code-of-conduct-languages/uk.md)
+- [वियतनामी/Tiếng Việt](code-of-conduct-languages/vi.md)
+  
+### सामुदायिक आचार संहिता
+CNCF समुदायातील योगदानकर्त्या(Contributors), देखभालकार्ययुक्त(Maintainers) आणि सहभागी(Participants) आपल्या पाळीत, आणि तसेच सर्वांचं मान-सम्मान करणाऱ्या उघड आणि स्वागतार्ह समुदायातील सर्व लोकांचं ध्येय आहे की, समस्यांची अहवाल (issues) करणे, सुविधा विनंत्या (feature requests) पोस्ट करणे, माहितीचंद्र (documentation) अद्ययावत करणे, पुल विनंत्या (PR) किंवा पॅच सबमिट करणे, सम्मेलनात किंवा कार्यक्रमात सहभाग घेणे, किंवा इतर समुदाय किंवा प्रकल्पाच्या क्रियाकलापांत सहभागी व्हावं.
```

Comment:
Your suggestion doesn't seem to completely fit the first sentence of the CoC. WDYT of the below suggestion?
```suggestion
CNCF समुदायातील योगदानकर्ते, देखरेख करणारे आणि सहभागी म्हणून, आणि खुल्या आणि स्वागतार्ह समुदायाला चालना देण्याच्या हितासाठी, आम्ही सर्व लोकांचा आदर करण्याचे वचन देतो जे समस्यांचे अहवाल देणे, वैशिष्ट्य विनंत्या पोस्ट करणे, दस्तऐवज अद्यतनित करणे, पुल विनंत्या किंवा पॅच सबमिट करणे, कॉन्फरन्स किंवा कार्यक्रमांना सहभाग घेणे किंवा इतर समुदाय किंवा प्रकल्प क्रियाकलापांमध्ये सहभागी होणे याद्वारे योगदान देतात किंवा सहभागी होतात.
```